### PR TITLE
Constants branch

### LINKED ISSUE #9: Replace all inline static occurences of default values for variables with static variables, which have to be defined in a separate interface named Constants
For example in such a case
```
.
.

            if(LABEL_TEXT_COLOR == null)
                LABEL_TEXT_COLOR = "#424242";
            if(VALUE_TEXT_COLOR == null)
.
.
```
Replace the static string "#424242" with a static string variable declared in a seperate interface by the name `Constants`

### PR DESCRIPTION
**Pull request by Sourish-Mukherjee**
**Objective**: Replace all inline static occurrences of default values for variables with static variables.

**Approach** : 

- Created an Interface named **Constants**
- Implemented **Constants** interface in **BarView** and **BarGroup** Class
- Modified **BarGroup** class, where I replaced all colour constant values with String variable name from Interface
- Modified **BarView** class, where I replaced all colour constant values with String variable name from Interface.

@krharsh17

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="JDK" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (krharsh17) [inline .idea/misc.xml]: This change is not needed

File: app/build.gradle [hunk 2]
```diff
@@ -22,6 +22,7 @@ android {
 dependencies {
     implementation project (":library")
     implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'com.github.krharsh17:barview-android:1.0'
```
Comment (krharsh17) [inline app/build.gradle]: Removing this will solve your CI build error.

File: library/src/main/java/in/krharsh17/barview/Constants.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package in.krharsh17.barview;
+
+public interface Constants {
+
+    String label_text_color = "#424242";
+    String value_text_color = "FFFFFF";
+    String ripple_color = "EEEEEE";
+    String charArray = "0123456789ABCDEF";
```
Comment (krharsh17) [inline library/src/main/java/in/krharsh17/barview/Constants.java]: Please use `camelCasing` in naming all variables throughout the project. For more details, refer to [this](https://en.wikipedia.org/wiki/Camel_case).

**Review by krharsh17 — CHANGES_REQUESTED** [review]
Good job on the task! Just make these changes and your work will be merged 😀

**Comment by Sourish-Mukherjee** [comment]
@krharsh17 Fixed The Issus, regarding project-jdk-name, my system always makes it 1.8 since I am using 1.8 JDK. Thus I forcefully edited it.

**Review by krharsh17 — APPROVED** [review]
Well done!

**Pull request merged**